README.md

```diff
@@ -1 +1,56 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard project 1
+
+## Description
+This is a basic website; a Leader board Application that displays a list that looks and behaves like the part of minimalist project. 
+
+
+## Getting started
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+Download the ZIP from this location, or run the following git command to clone the files to your machine:
+
+* Once the files are on your machine, open the Leaderboard folder in Visual Studio Code.
+
+* With the files open in Visual Studio Code, press the Go Live button at the bottom of the window to launch the files with Live Server.
+
```

Comment:
- [ ] in this part of the readme you mention that the user should use a git command to clone the project however it is never mentioned which command to use, try to explain how to clone and run the project locally as if the user is a novice.

Reply:
Asked change implemented